compiler/packages/babel-plugin-react-compiler/src/__tests__/fixtures/compiler/iife-return-modified-later-phi.expect.md

```diff
@@ -38,10 +38,11 @@ function Component(props) {
     items = t0;
 
     items?.push(props.a);
```

Comment:
nice, so we know that `props` is non-null and therefore can hoist the `props.a`, even though that property load is optional